docs/src/utils/environment.ts

```diff
@@ -0,0 +1,3 @@
+export const isDev = process.env.NODE_ENV === 'development';
+export const isProd = process.env.NODE_ENV === 'production';
+export const isTest = process.env.NODE_ENV === 'test';
```

Comment:
Looks like you added `FF_REACT_NATIVE` in https://github.com/aws-amplify/amplify-ui/pull/2354/files, where did we get `NODE_ENV` or it's some sort of default variable?

Reply:
The `isDev` flag no longer has relevancy to https://github.com/aws-amplify/amplify-ui/pull/2354 so i moved it in to its own PR. Added `isProd` and `isTest` so they would already be there when needed in the future

Reply:
For further context you can read [this comment](https://github.com/aws-amplify/amplify-ui/pull/2354#discussion_r931666984).